addons/mail/static/src/components/follow_button/follow_button.xml

```diff
@@ -3,9 +3,9 @@
 
     <t t-name="mail.FollowButton" owl="1">
         <t t-if="thread">
-            <div class="o_FollowButton" t-attf-class="{{ className }}" t-ref="root">
+            <div class="o_FollowButton" t-attf-class="{{ className }} {{ props.isChatterButton ? '' : 'o_ChatterTopbar_button' }}" t-ref="root">
```

Comment:
I think this is incorrect? You don't set `.o_ChatterTopbar_button` here because it's set on the `<button>` below, but when `FollowButton` is not in chatter, it doesn't make sense to set `o_ChatterTopbar_button` on root node.